examples/js/loaders/MMDLoader.js

```diff
@@ -427,7 +427,7 @@ THREE.MMDLoader.prototype.pourVmdIntoCamera = function ( camera, vmd, name ) {
 
 		var clip = new THREE.AnimationClip( name === undefined ? THREE.Math.generateUUID() : name, -1, tracks );
 
-		if ( clip !== null ) {
```

Comment:
I've checked the history.
I used `THREE.AnimationClip.parseAnimation()` which can return `null` before.
Seems like I've forgotten to remove `null` check logic when I replaced it with `new THREE.AnimationClip()`.